packages/cw-orch-contract-derive/src/lib.rs

```diff
@@ -1,18 +1,11 @@
 #![recursion_limit = "128"]
 
-mod cw_orch_contract;
-
-use crate::cw_orch_contract::{
-    get_crate_to_struct, get_func_type, get_response_generic_or_fallback, get_wasm_name,
-};
-
-use convert_case::{Case, Casing};
-use syn::{__private::TokenStream2, parse_macro_input, Fields, FnArg, GenericArgument, Item, Path};
+use syn::{__private::TokenStream2, parse_macro_input, Fields, GenericArgument, Item, Path};
```

Comment:
Hmmm should we be using this? `__private::TokenStream2`

Reply:
Yeah I know, but not sure how to do better